### PR TITLE
Feat(eos_cli_config_gen): Allow TCAM profile local file configuration

### PR DESCRIPTION
**Pull request by xaviramon**
## Change Summary

The current data model for TCAM profile allows only for a file parsing into the device configuration. With the new CLI keyword `source` we are able to read TCAM profiles from a local file. This change allows for a source file configuration in the data model. The file has to be present in the device before pushing the new configuration.

## Related Issue(s)

Fixes #<ISSUE ID>

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes
Expand data model to allow `source` keyword and remove mandatory field for `config` keyword.
```
tcam_profile:
  profiles:
    - name: MY_TEST
      source: "flash:/TCAM_PROFILES/MY_TEST.txt"
```

## How to test
Molecule included

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/tcam_profile.schema.yml [hunk 1]
```diff
@@ -28,7 +28,13 @@ keys:
...
+              required: false
               description: |
                 TCAM Profile Config. Since these can be very long, it is often a good idea to import the config from a file.
                 Example: "{{ lookup('file', 'TCAM_TRAFFIC_POLICY.conf') }}"
+            source:
+              type: str
+              required: false
+              description: |
+                TCAM profile local source path. Used to read the TCAM profile from a local path existing in the device.
+                The file can contain different TCAM profiles and the used will be defined by the "system" variable.
```
Comment (chetryan) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/tcam_profile.schema.yml]:
```suggestion
                The file can contain different TCAM profiles and the one used will be defined by the "system" variable.
```

**Review by chetryan — COMMENTED** [review]
LGTM apart from the typo in the description.

**Review by tgodaA — APPROVED** [review]
LGTM

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml [hunk 2]
```diff
@@ -11040,11 +11040,21 @@ keys:
...
+
+                '
+            source:
+              type: str
+              required: false
+              description: 'TCAM profile local source path. Used to read the TCAM
+                profile from a local path existing in the device.
+
+                The file can contain different TCAM profiles and the one used will
+                be defined by the "system" variable.'
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml]: Are we sure about this? The file is given under one profile, so how can it contain multiple profiles?

**Review by ClausHolbechArista — APPROVED** [review]

**Pull request merged**